============================= test session starts ==============================
tests/test_crypto.py ...........EE.....EE.................................................................s.s.......................................................................................................................FFs..........................F......................
tests/test_debug.py .
tests/test_rand.py ....
tests/test_ssl.py ............................................................s.ss..............................FF......................................FF.......s...........................s.............................................................
tests/test_tsafe.py .
tests/test_util.py .
==================================== ERRORS ====================================
=================================== FAILURES ===================================
tests/test_crypto.py:3196: AttributeError
tests/test_crypto.py:3219: AttributeError
tests/test_crypto.py:3623: Failed
tests/test_ssl.py:1616: 
tests/test_ssl.py:1625: 
tests/test_ssl.py:2433: 
tests/test_ssl.py:2445: 
========== 7 failed, 480 passed, 8 skipped, 4 error in x seconds ===========
